packages/kuma-gui/src/app/gateways/views/DelegatedGatewayDetailView.vue

```diff
@@ -12,50 +12,56 @@
     v-slot="{ can, route, t, me }"
   >
     <AppView>
-      <div class="stack">
+      <XLayout type="stack">
         <DataLoader
           :src="`/meshes/${route.params.mesh}/service-insights/${route.params.service}`"
           v-slot="{ data }: ServiceInsightSource"
         >
-          <KCard v-if="data">
-            <div class="columns">
-              <DefinitionCard>
-                <template #title>
-                  {{ t('http.api.property.status') }}
-                </template>
+          <XAboutSection
+            v-if="data"
+            :title="t('delegated-gateways.detail.about.title')"
+            :created="t('common.formats.datetime', { value: Date.parse(data.creationTime)})"
+            :modified="t('common.formats.datetime', { value: Date.parse(data.modificationTime)})"
+          >
+            <DefinitionCard layout="horizontal">
+              <template #title>
+                {{ t('http.api.property.status') }}
+              </template>
 
-                <template #body>
-                  <StatusBadge :status="data.status" />
-                </template>
-              </DefinitionCard>
+              <template #body>
+                <StatusBadge :status="data.status" />
+              </template>
+            </DefinitionCard>
 
-              <DefinitionCard>
-                <template #title>
-                  {{ t('http.api.property.address') }}
-                </template>
+            <DefinitionCard layout="horizontal">
+              <template #title>
+                {{ t('http.api.property.address') }}
+              </template>
 
-                <template #body>
-                  <TextWithCopyButton
-                    v-if="data.addressPort"
-                    :text="data.addressPort"
-                  />
+              <template #body>
+                <XCopyButton
+                  v-if="data.addressPort"
+                  variant="badge"
+                  format="default"
+                  :text="data.addressPort"
+                />
 
-                  <template v-else>
-                    {{ t('common.detail.none') }}
-                  </template>
+                <template v-else>
+                  {{ t('common.detail.none') }}
                 </template>
-              </DefinitionCard>
+              </template>
+            </DefinitionCard>
 
-              <ResourceStatus
-                :online="data.dataplanes?.online ?? 0"
-                :total="data.dataplanes?.total ?? 0"
-              >
-                <template #title>
-                  {{ t('http.api.property.dataPlaneProxies') }}
-                </template>
-              </ResourceStatus>
-            </div>
-          </KCard>
+            <ResourceStatus
+              layout="horizontal"
```

Comment:
Hmmmm 🤔 I am little confused now 😅 because:

1. Why is this not catched?
2. How does it work still work?

Regarding 2.: The default value of `layout` in `DefinitionCard` is `vertical`. Under the hood `ResourceStatus` uses `DefinitionCard`, without setting `layout`, because the default is `vertical` (what we want for `ResourceStatus`). However, when I set `layout="horizontal"` on the `ResourceStatus` it's somehow being passed down to the child `DefintionCard` 🤔

Reply:
I've just realized that I did the same with `XAboutSection`. I did not specify any prop, but since `AppAboutSection` is the top level component that wraps everything, I guess the props will be just passed further down there 🤔

Reply:
oh wait, sorry I didn't notice that `ResourceStatus` uses `DefinitionCard`

Because `DefinitionCard` is the first child of `ResourceStatus` it automatically gets another props pass to `ResourceStatus` passed down to it. SO this is why it still works and TS doesn't say anything.

I think this is still confusing though, I guess its a consequence of our other conversation about this horizontal style of ResourceStatus even being a thing.

Umm, not sure what to suggest here, if anything. 🤔 I think we can probably resolve this, tiny chance I might have another thought here, plus we might think of another way to do this (see other convo on the `Dataplanes: 0/20` thing)

Reply:
I'd resolve it then for now and revisit this once we discuss how we want to show the `ResourceDefinition` next to horizontal `DescriptionCards`.

Reply:
😆 all these different name forms are making my head spin!